KillerApp/src/main/java/com/example/killerapp/Manager.java

```diff
@@ -0,0 +1,154 @@
+package com.example.killerapp;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.util.Log;
+
+import com.dezen.riccardo.smshandler.ReceivedMessageListener;
+import com.dezen.riccardo.smshandler.SMSManager;
+import com.dezen.riccardo.smshandler.SMSMessage;
+import com.dezen.riccardo.smshandler.SMSPeer;
+
+/***
+ * @author Pardeep
+ * Class
+ */
+
+public class Manager {
+    private static Context currentContext;
+    AlarmManager alarmManager = new AlarmManager();
```

Comment:
Would be better to have these fields as private if you don't access them outside the class, and if you do access them outside the class, it should be done through a getter anyway